diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/orage-4.4.1/panel-plugin/Makefile.am src/orage-4.4.1/panel-plugin/Makefile.am
--- origsrc/orage-4.4.1/panel-plugin/Makefile.am	2007-01-25 14:49:52.000000000 -0600
+++ src/orage-4.4.1/panel-plugin/Makefile.am	2007-05-25 18:38:21.038589000 -0500
@@ -17,9 +17,7 @@
 if HAVE_CYGWIN
 orageclock_LDFLAGS +=                        	                \
 	-no-undefined                                                   \
-	@LIBX11_LDFLAGS@                                                \
-	@LIBXFCEGUI4_LIBS@                                              \
-	@LIBX11_LIBS@
+	@LIBXFCEGUI4_LIBS@
 endif
 
 # .desktop file
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/orage-4.4.1/plugin/Makefile.am src/orage-4.4.1/plugin/Makefile.am
--- origsrc/orage-4.4.1/plugin/Makefile.am	2007-01-25 14:49:52.000000000 -0600
+++ src/orage-4.4.1/plugin/Makefile.am	2007-05-25 18:02:35.485558800 -0500
@@ -7,6 +7,13 @@
 	-avoid-version							\
 	-module
 
+if HAVE_CYGWIN
+orage_plugin_la_LDFLAGS +=						\
+	-export-symbols $(datadir)/xfce4/devel/mcs-manager.def \
+	-no-undefined								\
+	$(XFCE_MCS_MANAGER_LIBS)
+endif
+
 orage_plugin_la_SOURCES =						\
 	xfcalendar_plugin.c
 
@@ -27,3 +34,4 @@
 DISTCLEANFILES =							\
 	$(desktop_DATA)
 
+
