--- origsrc/orage-4.4.1/panel-plugin/Makefile.am	2007-01-25 14:49:52.000000000 -0600
+++ src/orage-4.4.1/panel-plugin/Makefile.am	2007-05-25 18:38:21.038589000 -0500
@@ -17,9 +17,7 @@
 if HAVE_CYGWIN
 orageclock_LDFLAGS +=                        	                \
 	-no-undefined                                                   \
-	@LIBX11_LDFLAGS@                                                \
-	@LIBXFCEGUI4_LIBS@                                              \
-	@LIBX11_LIBS@
+	@LIBXFCEGUI4_LIBS@
 endif
 
 # .desktop file
--- origsrc/orage-4.4.1/plugin/Makefile.am	2007-01-25 14:49:52.000000000 -0600
+++ src/orage-4.4.1/plugin/Makefile.am	2007-05-25 18:02:35.485558800 -0500
@@ -7,6 +7,13 @@
 	-avoid-version							\
 	-module
 
+if HAVE_CYGWIN
+orage_plugin_la_LDFLAGS +=						\
+	-export-symbols $(datadir)/xfce4/devel/mcs-manager.def \
+	-no-undefined								\
+	$(XFCE_MCS_MANAGER_LIBS)
+endif
+
 orage_plugin_la_SOURCES =						\
 	xfcalendar_plugin.c
 
@@ -27,3 +34,4 @@
 DISTCLEANFILES =							\
 	$(desktop_DATA)
 
+
